doc/start/intro.rst

```diff
@@ -2,14 +2,19 @@
  Intro to Ceph
 ===============
 
-Whether you want to provide :term:`Ceph Object Storage` and/or
-:term:`Ceph Block Device` services to :term:`Cloud Platforms`, deploy
-a :term:`Ceph File System` or use Ceph for another purpose, all
-:term:`Ceph Storage Cluster` deployments begin with setting up each
-:term:`Ceph Node`, your network, and the Ceph Storage Cluster. A Ceph
-Storage Cluster requires at least one Ceph Monitor, Ceph Manager, and
-Ceph OSD (Object Storage Daemon). The Ceph Metadata Server is also
-required when running Ceph File System clients.
+Ceph can be used to provide :term:`Ceph Object Storage` to :term:`Cloud
+Platforms` and Ceph can be used to provide :term:`Ceph Block Device` services
+to :term:`Cloud Platforms`. Ceph can be used to deploy a :term:`Ceph File
```

Comment:
There are also direct librados clients but they are so few that we need not mention that here.

Reply:
Eventually we'll have a section on custom-built librados clients. It's on a list.

Reply:
I only recall having seen mention of a total of *2* so probably anyone who needs to know already does.